config/database.go

```diff
@@ -49,3 +58,11 @@ func DbURL(dbConfig *DBConfig) string {
 		dbConfig.DBName,
 	)
 }
+
+func (dbsql *DBMySQL) MysqlConn() *gorm.DB {
+	result, err := gorm.Open(mysql.Open(dbsql.DbURL(dbsql.BuildDBConfig())), &gorm.Config{})
+	if err != nil {
+		return nil
```

Comment:
Why it's returning nil? :confused:

Reply:
As long the open statement suceed, they will return a connection. Then, return error when open statement has failed.

Reply:
Why not using panic then? so we can know the error message

Reply:
There's nothing wrong with it. We also can use that too.